content/en/llm_observability/exploration_and_correlation/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Exploration and Correlation
+description: Learn about how to explore more of your application in LLM Observability.
+---
+
+## Overview
+
+Explore and analyze your LLM applications in production with tools for querying, visualizing, correlating, and investigating data across traces, clusters, and beyond.
+
+Monitor performance, debug issues, evaluate quality, and secure your LLM-powered systems with unified visibility across traces, metrics, and online evaluations.
+
+### Real-time Performance Monitoring
+
+Monitor your LLM application's operational health with built-in metrics and dashboards:
+
+{{< img src="llm_observability/index/llm_dashboard.png" alt=" This image shows a built‑in dashboard highlighting LLM Observability metrics." style="width:100%">}}
+
+- Request volume and latency: Track requests per second, response times, and performance bottlenecks across different models, operations, and endpoints.
+- Error tracking: Monitor HTTP errors, model timeouts, and failed requests with detailed error context.
+- Token consumption: Track prompt tokens, cached tokens, completion tokens, and total usage to optimize costs.
+- Model usage analytics: Monitor which models are being called, their frequency, and performance characteristics.
+
+The out-of-the-box LLM Overview Dashboard provides consolidated views of trace-level and span-level metrics, error rates, latency breakdowns, token consumption trends, and triggered monitors.
```

Comment:
Let's link out to: https://app.datadoghq.com/dash/integration/llm_operational_insights?fromUser=false&refresh_mode=sliding&from_ts=1758905575629&to_ts=1758909175629&live=true